rest-service/manager_rest/test/infrastructure/test_list_filters.py

```diff
@@ -276,7 +275,6 @@ def test_blueprints_list_with_filters(self):
                                       'values {0}, got {1}'
                                       .format(filter_params, blueprint))
         self.assertEqual(self.first_blueprint_id, blueprint['id'])
-        self.assertIsNotNone(response[0]['plan'])
```

Comment:
Why did you remove these checks and/or why were we asserting them before? They look unrelated to anything this test needs to test?

Reply:
oh yeah, forgot to mention. Indeed I'm not going to check that plan specifically is set. We check that the right blueprint are returned by the filters, I dont see much reason to check for plan specifically. And in this case it is None becaue I don't mock it. I could but why.